site/src/App/foundations/layout/layout.tsx

```diff
@@ -0,0 +1,512 @@
+
+type Space = 'none' | keyof typeof tokens.space;
+const spaceScale = ['none', ...Object.keys(tokens.space)] as Space[];
+
+const lipsum1 =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse dignissim dapibus elit, vel egestas felis pharetra non. Cras malesuada, massa nec ultricies efficitur, lectus ante consequat magna, a porttitor massa ex ut quam.';
+
+const lipsum2 =
+  'Phasellus ipsum tortor, aliquet dapibus fermentum in, mollis vel metus. Vestibulum malesuada ante eu velit malesuada, nec ultricies sapien finibus. Aenean rutrum in sem a ullamcorper. Integer ut euismod urna. Interdum et malesuada fames ac ante ipsum primis in faucibus.';
+
+const page: Page = {
+  title: 'Layout',
+  Component: () => {
+    return (
+      <TextStack>
+        <Heading level="2">Layout</Heading>
+        <Text>
+          The guiding principle for layout in Braid is that components should
+          not provide surrounding white space. Instead, spacing between elements
+          is owned entirely by layout components. This approach ensures that the
+          system is as composable as possible while keeping white space
+          completely predictable.
+        </Text>
+        <Text>
+          All of the components referenced below can be infinitely nested within
+          each other to create a wide variety of standard layouts. Getting a
+          firm grasp of these components is an essential part of working
+          effectively with Braid.
+        </Text>
+
+        <Divider />
+
+        <Heading level="3">Spacing</Heading>
+        <Text>
+          Braid provides a standard white space scale that is available across
+          the entire component suite. As much as possible, Braid tries to make
+          use of this scale rather than generating custom spacing rules.
+        </Text>
+
+        <Stack space="small">
+          {spaceScale.map(space => (
+            <Columns space="xsmall" alignY="center">
+              <Column width="content">
+                <Box style={{ width: 100 }}>
+                  <Text baseline={false}>{space}</Text>
+                </Box>
+              </Column>
+              <Column>
+                <Box display="flex">
+                  <Box
+                    background="formAccent"
+                    paddingLeft={space}
+                    paddingTop="medium"
+                  />
+                </Box>
+              </Column>
+            </Columns>
+          ))}
+        </Stack>
+
+        <Text tone="secondary" size="small">
+          Note: The exact pixel values in this scale may vary betweeen themes.
+        </Text>
+
+        <Divider />
+
+        <Heading level="3">Box</Heading>
+        <Text>
+          Box is the most low-level layout component provided by Braid. Its job
+          is to render an individual element on the screen.
+        </Text>
+        <Text>
+          In terms of page layout, Box most notably provides a set of padding
```

Comment:
Might be worth a sweep around consistently wrapping component names in backticks, eg. \`Box\`